common/src/api/schema.ts

```diff
@@ -99,14 +99,15 @@ export const API = (_apiTypeCheck = {
       })
       .strict(),
   },
-  'cancel-bet': {
+  'bet/cancel/:betId': {
```

Comment:
It's OK to use the Express named parameter syntax for this, I would probably bother to typecheck it though, because the whole thing is getting abstract enough that it might start getting painful to debug any errors.